lib/image-stream/handlers.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
       return Bluebird.props({
         path: imagePath,
         archiveExtension: fileExtensions.getLastFileExtension(imagePath),
-        extension: fileExtensions.getPenultimateFileExtension(imagePath),
+        extension: fileExtensions.getPenultimateFileExtension(imagePath) || 'img',
```

Comment:
Could we change `getPenultimateFileExtension` itself to fallback to `img`?

Reply:
I think that'd cause the archive handling to not work as expected, but would need to double-check. This whole file extension handling needs a rework imo, while this is just a quick fix for some distros that omit the secondary extension for some reason.

Reply:
Also maybe we could make `'img'` into a constant `DEFAULT_EXT` or something along those lines?

Reply:
@Shou yup, definitely – updated!

Reply:
> some distros that omit the secondary extension

Just out of curiosity, could you link to some examples? (just as a comment here, not as a code-comment in the PR)